client/my-sites/plugins/plugin-item/plugin-item.jsx

```diff
@@ -186,21 +195,26 @@ module.exports = React.createClass( {
 	},
 
 	renderActions() {
+		const {
+			activation: canToggleActivation,
+			autoupdate: canToggleAutoupdate,
+		} = this.props.allowedActions;
+
 		return (
 			<div className="plugin-item__actions">
-				<PluginActivateToggle
+			{ canToggleActivation && <PluginActivateToggle
```

Comment:
I wonder if we could hide these components in the same way that they are disabled. Instead of ...

```es6
{ canToggleAutoupdate && <PluginAutoupdateToggle
  isMock={ this.props.isMock }
  // ...
  /> }
}
```

... implement something like ...

```es6
<PluginAutoupdateToggle
  isMock={ this.props.isMock }
  hide={ ! canToggleAutoupdate }
  // ...
/>
```

I think it looks more readable, but I don't wanna bothering you with too many details. Again, feel free to ignore this.